wsd/wopi/WopiStorage.cpp

```diff
@@ -806,9 +807,6 @@ void WopiStorage::uploadLocalFileToStorageAsync(const Authorization& auth, LockC
                         << ex.what());
         _uploadHttpSession.reset();
     }
-
-    scopedInvokeCallback.setArg(AsyncUpload(
```

Comment:
Don't we want to call here the callback with failed state? it will happen on exceptions

Reply:
yup